tests/test_component_mapping_fix.py

```diff
@@ -0,0 +1,49 @@
+import math
```

Comment:
Please merge this test into `test_data.py` instead. Currently, there is only one file with tests for every module. I guess we should change that on the long run, but until then I'd stick to the old scheme.